node_modules/oae-tenants/lib/api.js

```diff
@@ -800,20 +814,22 @@ var _setLandingPageBlockAttribute = function(ctx, block, blockName, attributeNam
  * `throw` anything.
  *
  * @param  {Validator}  validator               The Validator object
- * @param  {String}     emailDomain             The email domain expression to verify
+ * @param  {String[]}   emailDomains            The email domain expressions to verify
  * @param  {String}     [updateTenantAlias]     The alias of the tenant being updated, if any
  * @api private
  */
-var _validateEmailDomain = function(validator, emailDomain, updateTenantAlias) {
-    var matchingTenantAlias = tenantEmailDomainIndex.conflict(updateTenantAlias, emailDomain);
-    var matchingTenant = tenants[matchingTenantAlias];
-    var matchingEmailDomain = (matchingTenant && matchingTenant.emailDomain);
-
-    var err = {
-        'code': 400,
-        'msg': util.format('The email domain "%s" conflicts with existing email domain: %s', emailDomain, matchingEmailDomain)
-    };
-    validator.check(matchingEmailDomain, err).isNull();
+var _validateEmailDomains = function(validator, emailDomains, updateTenantAlias) {
+    _.each(emailDomains, function(emailDomain) {
+        var matchingTenantAlias = tenantEmailDomainIndex.conflict(updateTenantAlias, emailDomain);
+        var matchingTenant = tenants[matchingTenantAlias];
+        var matchingEmailDomains = (matchingTenant && matchingTenant.emailDomains);
+
+        var err = {
+            'code': 400,
+            'msg': util.format('The email domain "%s" conflicts with existing email domains: %s', emailDomain, matchingEmailDomains)
+        };
+        validator.check(matchingEmailDomains, err).isNull();
```

Comment:
Should this `isNull` change to something that's more appropriate for an array that could be empty?